files/zh-tw/web/javascript/guide/grammar_and_types/index.md

```diff
@@ -530,7 +530,7 @@ var re = /ab+c/;
 
 ### 字串字面值 (String literals)
 
-字串字面值是用雙引號（“）或單引號（'）包住的零或多個字元。字串必須用同類的引號定界；也就是「兩個單引號」或「兩個雙引號」。以下是字串字面值的範例：
+字串字面值是用雙引號（「）或單引號（'）包住的零或多個字元。字串必須用同類的引號定界；也就是「兩個單引號」或「兩個雙引號」。以下是字串字面值的範例：
```

Comment:
string literal should use `"` rather than `“`, this should be a mistake

```suggestion
字串字面值是用雙引號（"）或單引號（'）包住的零或多個字元。字串必須用同類的引號定界；也就是「兩個單引號」或「兩個雙引號」。以下是字串字面值的範例：
```